source/includes/photos/_update.md

```diff
@@ -0,0 +1,68 @@
+## Update A Photo
+
+> `PUT http://localhost:3000/api/v6/photos/73f5f971-c6d6-4495-b2bf-022380d27c23`
+
+```ruby
+example_params = {
+  file: [String]
+}
+```
+
+> Example response
+
+```json
+{
+  "item": {
+    "id": 28,
```

Comment:
json should really be 'photo': {}, not 'item': {}. so either change in api and app, or leave it like this

Reply:
probably a @pacMakaveli question as there is likely an App based reason as to why it's returning a photo in an item hash